### DISCUSSION TITLE
Wikimedia uses tegola for their new maps

### DISCUSSION THREAD

**Discussion by ARolek** [go-spatial/tegola#851] [Show and tell]
We have the great pleasure working with the maps team at Wikimedia to deploy tegola. Here's a great post they released about the project: https://techblog.wikimedia.org/2022/05/06/modernizing-our-tech-stack-for-serving-maps-at-wikipedia/

**Comment by kszafran** [comment]
That's great news! I'm happy to see Golang in general and Tegola specifically gaining traction in the GIS community.

**Comment by meilinger** [comment]
Congratulations!  This is awesome news!
